### ISSUE TITLE
BooruSlideshow just doesn't work with newer versions of Electron.

### ISSUE THREAD

**Issue by SpotPot** [DontTalkToMeThx/BooruSlideshowElectron#10]
Even with the new "fix" Slideshow just doesn't work.
I'm a complete scrub when it comes to this stuff but I've browsed for about 3 days straight on to fix this to avail.
npm audit fix --force doesn't solve the issue either.
![electron_PxDcR5XEd8](https://github.com/DontTalkToMeThx/BooruSlideshowElectron/assets/85971862/d06bc4d8-0d4d-4c56-bff7-5fc4b9d845f0)
![cmd_L0Bq6B6QTM](https://github.com/DontTalkToMeThx/BooruSlideshowElectron/assets/85971862/f425328b-fb7f-4427-8f32-9fca1a595319)

**Comment by toufuguy** [comment]
Yep, just want to say that I'm having the same issue here with that error log - electron-download-manager can't summon `require("electron")` properly. (Image is of electron-download-manager internals.)

![image](https://github.com/DontTalkToMeThx/BooruSlideshowElectron/assets/54731958/84221c85-0f5b-4792-9d56-66c9145192af)

I just want to say that greatly appreciate all you've done maintaining this application so far, and this is out of my wheelhouse for debugging further, I wish you the best
